### ISSUE TITLE
Incorrect python version error when trying to compile test bas program

### ISSUE THREAD

**Issue by anatoliyv** [Dotneteer/kliveide#569]
I'm trying to launch default project files on macOS v.13.5.1

`.bas` file output:
```
Compiling /Users/hazza/Downloads/Test2/code/code.zxb.asm
Compilation failed: The process exited with code 1. Error: Command failed: /Users/hazza/Downloads/zxbasic/zxbasm /Users/hazza/Downloads/Test2/code/code.zxb.asm --output /Users/hazza/Downloads/Test2/code/code.zxb.asm.bin --optimize 2 -d /Users/hazza/Downloads/zxbasic/zxbasm.py Error: minimum Python Version required is 3.10
```

Similar error is for `.asm` file
```
Compiling /Users/hazza/Downloads/Test2/code/code.zxb.asm
Compilation failed: The process exited with code 1. Error: Command failed: /Users/hazza/Downloads/zxbasic/zxbasm /Users/hazza/Downloads/Test2/code/code.zxb.asm --output /Users/hazza/Downloads/Test2/code/code.zxb.asm.bin --optimize 2 -d /Users/hazza/Downloads/zxbasic/zxbasm.py Error: minimum Python Version required is 3.10
```

On my machine I have following:
```
❯ python3 --version
Python 3.11.5
```

Btw, launching commands locally seems working correct
```
❯ /Users/hazza/Downloads/zxbasic/zxbc /Users/hazza/Downloads/Test2/code/program.zxbas --output /Users/hazza/Downloads/Test2/code/program.zxbas.bin --optimize 2 --org 32768 --heap-size 4096

❯ echo $?
0
```

**Comment by vrishe** [comment]
Hello, @anatoliyv !

The error you're getting comes from ZXBasic's root package [initialization script](https://github.com/boriel/zxbasic/blob/master/src/__init__.py). Feels like something is happening when a child process is [being spawned](https://github.com/Dotneteer/kliveide/blob/master/src/main/compiler-integration/CompilerBase.ts#L73) by Klive.

May it be possible that you have another python installation? I would start with a little modification to [__init__.py](https://github.com/boriel/zxbasic/blob/master/src/__init__.py) like: `print(sys.executable, sys.version_info)` to see which python version it sees when run from Klive. In addition it is interesting to see the result of `which python3` from yours local shell.

**Comment by Dotneteer** [comment]
Fixed in version [v0.30.3](https://github.com/Dotneteer/kliveide/releases/tag/v0.30.3) :-)
